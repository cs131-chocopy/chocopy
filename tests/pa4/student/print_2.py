a:object = None
b:[object] = None
e:object=None
f:int =1
c:bool =True
d:str= "114514"
def getx(c:[object]) -> object:
    return c[0]
a = [[1,2],[2,3]]
b = [1,None,[1,2]]
print(getx(b))
e=f
print(e)
e=False if True else object()
print(e)

