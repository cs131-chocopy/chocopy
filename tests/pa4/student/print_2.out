1
1
False
